TikTokApi/tiktok.py

```diff
@@ -160,14 +169,22 @@ def getData(self, **kwargs) -> dict:
             proxies=self.__format_proxy(proxy),
         )
         try:
+            json = r.json()
+            if "type" in json and json["type"] == "verify" :
+                logging.error("Tiktok wants to display a catcha. Response is:\n" + r.text)
+                raise TikTokCaptchError()
             return r.json()
-        except Exception as e:
-            logging.error(e)
-            logging.error(
-                "Converting response to JSON failed response is below (probably empty)"
-            )
-            logging.info(r.text)
-            raise Exception("Invalid Response")
+        except JSONDecodeError as e:
+            text = r.text
+            logging.error("TikTok response: " + text)
+            if len(text) == 0 :
+                raise Exception("Empty response from Tiktok to " + url) from None
+            else :
+                logging.error(
+                    "Converting response to JSON failed response is below (probably empty)"
+                )
+                logging.error(e)
+                raise Exception("Invalid Response") from None
```

Comment:
```suggestion
                raise Exception("Invalid Response") from e
```
There isn't much value in raising `from None`.  Either omit the `from` clause, or raise from the actual exception (preferred)